mail_tracking/models/mail_message.py

```diff
@@ -253,14 +253,9 @@ def _prepare_dict_failed_message(self):
             return
         failed_partners = failed_trackings.mapped("partner_id")
         failed_recipients = failed_partners.name_get()
-        if self.author_id:
-            author = self.author_id.name_get()[0]
-        else:
-            author = (-1, _("-Unknown Author-"))
```

Comment:
Why removing this?

Reply:
Hello @chienandalu, sorry for the delay!

I deleted these lines because I'm using the same JS model for failed messages and "normal" messages: the `Message` model - as opposed to `v15` in which a new JS model was created for failed messages. This allows us to reuse the same components for the failed messages (or inheriting them and tweaking them a bit, as I did in this PR).

Since the `Message` model already has an `author` field referencing the `Partner` model, and this other `author` is a string, the fields would collide because of a type mismatch and raise an error. The `authorName` field already exists on the `Message` model which basically achieves the same result, so it seems this other `author` field is no longer needed, but what do you think? :thinking: